src/main/java/com/code_intelligence/jazzer/junit/DictionaryFile.java

```diff
@@ -23,13 +23,13 @@
 import java.lang.annotation.Target;
 
 /**
- * Defines a reference to a dictionary within the resources directory. The given {@code
- * resourcePath} must be an absolute path and must not have a leading {@code /}. These should follow
- * <a href="https://llvm.org/docs/LibFuzzer.html#dictionaries">libfuzzer's dictionary syntax</a>.
+ * Attaches a dictionary file located in resources to a fuzz test. The file should follow <a
+ * href="https://llvm.org/docs/LibFuzzer.html#dictionaries">libfuzzer's dictionary syntax</a>.
  */
 @Target({ElementType.METHOD, ElementType.ANNOTATION_TYPE})
 @Retention(RetentionPolicy.RUNTIME)
 @Repeatable(DictionaryFiles.class)
 public @interface DictionaryFile {
+  /** The resource path of the dictionary file. */
   String resourcePath();
```

Comment:
Should this also be changed to `value` so you don't need to include `resourcePath`?

Reply:
I thought about that, but found `@DictionaryFile(resourcePath = 'foo')` to be more descriptive. It also makes it easier for us to introduce `filePath` or a similar element in case we should need to later.